packages/date-picker/test/keyboard-input.test.js

```diff
@@ -484,37 +485,92 @@ describe('keyboard', () => {
       expect(result.getFullYear()).to.equal(99);
     });
 
-    it('should parse three digits with short year', async () => {
-      await sendKeys({ type: '6/20/99' });
-      const result = focusedDate();
-      expect(result.getFullYear()).to.equal(1999);
-    });
-
-    it('should parse three digits with short year 2', async () => {
-      await sendKeys({ type: '6/20/20' });
-      const result = focusedDate();
-      expect(result.getFullYear()).to.equal(2020);
-    });
-
-    it('should parse three digits with short year 3', async () => {
-      await sendKeys({ type: '6/20/1' });
-      const result = focusedDate();
-      expect(result.getFullYear()).to.equal(2001);
-    });
-
     it('should parse three digits with negative year', async () => {
       await sendKeys({ type: '6/20/-1' });
       const result = focusedDate();
       expect(result.getFullYear()).to.equal(-1);
     });
 
     it('should parse in base 10', async () => {
+      datepicker.referenceDate = new Date(2022, 0, 1);
```

Comment:
`referenceDate` needs to be set on the I18N object.

Reply:
Addressed in the latest commit